src/network/NetworkController.ts

```diff
@@ -99,6 +99,8 @@ export class NetworkController extends BaseController<NetworkConfig, NetworkStat
       case 'rpc':
         rpcTarget && this.setupStandardProvider(rpcTarget, chainId, ticker, nickname);
         break;
+      default:
+        throw new Error(`Unrecognized network type: '${type}'`);
```

Comment:
This is a functional change. Previously it would do nothing if the type was unrecognized.